cart/templates/cart/cart.html

```diff
@@ -1 +1,9 @@
-<h1>Cart</h1>
\ No newline at end of file
+<h1>Cart</h1> 
+<ul>
+    {% for object in object_list %}
+        <p>Object name: {{object.name}}</p>
+        <p>Description: {{object.description}}</p>
```

Comment:
<p>Product name: {{object.product.name}}</p>
    <p>Quantity: {{object.quantity}}</p>

Reply:
cart model does not have a description field.